src/models/user

```diff
@@ -0,0 +1,218 @@
+const mongoose = require('mongoose')
+
+const cities = ['Adana', 'Kocaeli', 'Adiyaman', 'Konya'
+    , 'Afyonkarahisar', 'Kutahya'
+    , 'Agri', 'Malatya'
+    , 'Amasya', 'Manisa'
+    , 'Ankara', 'Kahramanmaras'
+    , 'Antalya', 'Mardin'
+    , 'Artviin', 'Mugla'
+    , 'Aydin', 'Mus'
+    , 'Balikesir', 'Nevsehir'
+    , 'Bilecik', 'Nigde'
+    , 'Bingol', 'Ordu'
+    , 'Bitlis', 'Rize'
+    , 'Bolu', 'Sakarya'
+    , 'Burdur', 'Samsun'
+    , 'Bursa', 'Siirt'
+    , 'Canakkale', 'Sinop'
+    , 'Cankiri', 'Sivas'
+    , 'Corum', 'Tekirdag'
+    , 'Denizli', 'Tokat'
+    , 'Diyarbakir', 'Trabzon'
+    , 'Edirne', 'Tunceli'
+    , 'Elazig', 'Sanliurfa'
+    , 'Erzincan', 'Usak'
+    , 'Erzurum', 'Van'
+    , 'Eskisehir', 'Yozgat'
+    , 'Gaziantep', 'Zonguldak'
+    , 'Giresun', 'Aksaray'
+    , 'Gumushane', 'Bayburt'
+    , 'Hakkari', 'Karaman'
+    , 'Hatay', 'Kirikkale'
+    , 'Isparta', 'Batman'
+    , 'Mersin', 'Sirnak'
+    , 'Istanbul', 'Bartin'
+    , 'Izmir', 'Ardahan'
+    , 'Kars', 'Igdir'
+    , 'Kastamonu', 'Yalova'
+    , 'Kayseri', 'Karabuk'
+    , 'Kirklareli', 'Kilis'
+    , 'Kirsehir', 'Osmaniye'
+    , 'Duzce'
+]
+
+const listOfCategories = ['No Poverty', 'Zero Hunger', 'Good Health And Well-Being',
+    'Quality Education', 'Gender Equality', 'Clean Water And Sanitation',
+    'Affordable And Clean Energy', 'Animals', 'Oceans', 'Nature',
+    'Reduced Inequalities', 'Sustainable Cities And Communities', 'Responsible Consumption And Production',
+    'Climate Action', 'Life Below Water', 'Life On Land', 'Peace', 'Youth', 'Justice']
+
+const rate = mongoose.Schema({
+    userId: {
+        type: mongoose.Schema.Types.ObjectId,
+        ref: "User",
+        required: true
+    },
+    rate: {
+        type: number,
+        enum: [1, 2, 3, 4, 5],
+        required: true
+    }
+})
+
+const baseUser = mongoose.Schema({
+    email: {
+        type: String,
+        unique: true,
+        required: true
+    },
+    hashedPassword: {
+        type: String,
+        required: true
+    },
+    provider: {
+        type: String,
+        enum: ['email', 'google', 'twitter'],
+        default: 'email',
+        required: true
```

Comment:
I dont think this should be required since there will be a local startgey no? @muhammed-shihebi

Reply:
I think the local strategy is `email`